ffmpeg/ffmpeg_test.go

```diff
@@ -1512,3 +1512,60 @@ func TestTranscoder_FormatOptions(t *testing.T) {
 		t.Error("Did not get expected error with invalid format ", err)
 	}
 }
+
+// test short segments
+func shortSegments(t *testing.T, accel Acceleration, fc int) {
```

Comment:
I've been putting tests using the "new API" (NewTranscoder / Transcode / StopTranscoder) under `api_test.go` partially because this file was getting large. But up to you.